include/mbedtls/md.h

```diff
@@ -158,6 +161,14 @@ typedef enum {
     MBEDTLS_MD_SHA3_512,    /**< The SHA3-512 message digest. */
 } mbedtls_md_type_t;
 
+/* Note: this should always be >= PSA_HASH_MAX_SIZE
+ * in all builds with both CRYPTO_C and MD_LIGHT.
```

Comment:
This looks like a good candidate for a static assert (even a manual one with pre-processor checks along the lines of `#if defined(XX) && XXX < PSA_HASH_MAX_SIZE ... #error`)

Reply:
We have one [here](https://github.com/Mbed-TLS/mbedtls/blob/development/library/md.c#L74), thanks to Gilles making the same remark when I first added this comment :)

Reply:
In which case this comment needs to be change, since we know it is true

`/* Note: this will always be >= PSA_HASH_MAX_SIZE`